src/index.js

```diff
@@ -105,16 +126,90 @@ function startTurn () {
   turn.resetTimer()
   let guess = turn.evaluateGuess($('#submit-form__answer-input').val())
   round.removeAnswer(guess, turn.player)
-  console.log(turn.player)
   hilightPlayer()
+  updatePlayerScore()
 }
 
 function hilightPlayer() {
   if (turn.player.id === 1) {
     $('.p1__box').removeClass('current-player')
     $('.p2__box').addClass('current-player')
+    hideTimer(2)
   } else {
     $('.p2__box').removeClass('current-player')
     $('.p1__box').addClass('current-player')
+    hideTimer(1)
   }
 }
+
+function hideTimer(index) {
+  if (index === 1) {
+    $('.timer-1').parent().removeClass('hidden')
+    $('.timer-2').parent().addClass('hidden')
+  } else {
+    $('.timer-2').parent().removeClass('hidden')
+    $('.timer-1').parent().addClass('hidden')
+  }
+}
+
+
+function updatePlayerScore() {
+  $('#score-box__player-1-score').text(game.player1.score)
+  $('#score-box__player-2-score').text(game.player2.score)
+}
+
+function changeRound() {
+  if ((round.answers.length === 0) && (game.round < 2)) {
+    survey.randomizeSurveys()
```

Comment:
The first three lines in this if/else seem to be repeated. You can move them outside of the if/else since they run for both cases.